packages/rest/src/utils/response.ts

```diff
@@ -0,0 +1,16 @@
+import type { ApiError } from '../types'
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export function alternativeResponse<T>(input: T): any {
```

Comment:
`T` can be return type as well right?

Reply:
No as then the return type on the method is wrong. This is my workaround to have alternative responses in TSOA. The proper way to do it (inject error handler) doesn't allow you to set examples for alternative status codes.

This way the OpenAPI / swagger is nice, which is most important to me